### PR TITLE
Add Habana Gaudi 2 blog post

### PR DESCRIPTION
**Pull request by regisss**
This blog post presents how to use Gaudi 2 now and the speedup it provides against Gaudi 1 and A100.

### CODE REVIEW

File: _blog.yml [hunk 1]
```diff
@@ -1522,3 +1522,12 @@
   tags:
     - guide
     - fine-tuning
+
+- local: habana-gaudi-2-benchmark
+  title: "Faster Training and Inference: Habana Gaudi 2 vs Nvidia A100 80GB"
+  author: regisss
+  thumbnail: /blog/assets/habana-gaudi-2-benchmark/thumbnail.png
```
Comment (julien-c) [inline _blog.yml:1529]: thumbnail image is not the right aspect ratio, there are some specs in the README i think

File: _blog.yml [hunk 2]
```diff
@@ -1522,3 +1522,12 @@
...
+  date: December 09, 2022
```
Comment (julien-c) [inline _blog.yml:1530]:
```suggestion
  date: December 9, 2022
```

File: habana-gaudi-2-benchmark.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+*BS* is the batch size per device. Gaudi 2 and A100 runs were performed in fp32 with gradient checkpointing enabled.
+
+
+## Conclusion
+
+In this article, you have seen that Habana Gaudi 2 significantly improves Gaudi 1 and is about twice as fast as Nvidia A100 80GB for both training and inference.
+
+You also know now how to setup a Gaudi 2 instance through the Intel Developer Zone. Check out the [examples](https://github.com/huggingface/optimum-habana/tree/main/examples) you can easily run on such a device with 🤗 Optimum Habana.
+
+If you are interested in accelerating your training and/or inference workflows, [contact our experts](mailto:expert-acceleration@huggingface.co) to learn about our [Expert Acceleration Program](https://huggingface.co/support). To learn more about Habana solutions, [read about our partnership and how to contact them](https://huggingface.co/hardware/habana).
```
Comment (julien-c) [inline habana-gaudi-2-benchmark.md:140]: maybe also link to the Hardware program?

File: habana-gaudi-2-benchmark.md [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+</div>
+
+<div class="author-card">
+    <a href="/regisss">
+        <img class="avatar avatar-user" src="https://aeiljuispo.cloudimg.io/v7/https://s3.amazonaws.com/moonup/production/uploads/1644920200150-620b7c408f5871b8a1a168a7.jpeg?w=200&h=200&f=face" title="Gravatar">
+        <div class="bfc">
+            <code>regisss</code>
+            <span class="fullname">Régis Pierrard</span>
+        </div>
+    </a>
```
Comment (julien-c) [inline habana-gaudi-2-benchmark.md:22]: Maybe you can have a co-author from Habana?

**Review by julien-c — COMMENTED** [review]
left some nits but looks good to me! cool subject

File: habana-gaudi-2-benchmark.md (hunk 4 above)
Comment (regisss) [reply habana-gaudi-2-benchmark.md:22]: @sreeganesan Any idea who could be a co-author on your side?

File: habana-gaudi-2-benchmark.md [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+<figure class="image table text-center m-0 w-full">
+  <img src="assets/habana-gaudi-2-benchmark/launch_instance.png" alt="Cloud Architecture"/>
+</figure>
+
+4. You can then request an instance:
+<figure class="image table text-center m-0 w-full">
+  <img src="assets/habana-gaudi-2-benchmark/request_instance.png" alt="Cloud Architecture"/>
+</figure>
+
+5. Once your request is validated, re-do the step 3 and click on *Add OpenSSH Publickey* to add a payment method (credit card or promotion code) and a SSH publick key that you can generate with `ssh-keygen -t rsa -b 4096 -f ~/.ssh/id_rsa`. You may be redirected to step 3 each time you add a payment method or a SSH public key.
```
Comment (jeffboudier) [inline habana-gaudi-2-benchmark.md:50]:
```suggestion
5. Once your request is validated, re-do the step 3 and click on *Add OpenSSH Publickey* to add a payment method (credit card or promotion code) and a SSH public key that you can generate with `ssh-keygen -t rsa -b 4096 -f ~/.ssh/id_rsa`. You may be redirected to step 3 each time you add a payment method or a SSH public key.
```

File: habana-gaudi-2-benchmark.md [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+*BS* is the batch size per device. Gaudi 2 and A100 runs were performed in fp32 with gradient checkpointing enabled.
+
+
+## Conclusion
+
+In this article, you have seen that Habana Gaudi 2 significantly improves Gaudi 1 and is about twice as fast as Nvidia A100 80GB for both training and inference.
+
+You also know now how to setup a Gaudi 2 instance through the Intel Developer Zone. Check out the [examples](https://github.com/huggingface/optimum-habana/tree/main/examples) you can easily run on such a device with 🤗 Optimum Habana.
+
+If you are interested in accelerating your training and/or inference workflows, contact us to learn about our [Hardware Partner Program](https://huggingface.co/hardware) and our [Expert Acceleration Program](https://huggingface.co/support). To learn more about Habana solutions, [read about our partnership and how to contact them](https://huggingface.co/hardware/habana).
```
Comment (jeffboudier) [inline habana-gaudi-2-benchmark.md:140]:
```suggestion
If you are interested in accelerating your Machine Learning training and inference workflows using the latest AI hardware accelerators and software libraries, check out our [Expert Acceleration Program](https://huggingface.co/support). To learn more about Habana solutions and contact Habana, [you can read about our partnership here](https://huggingface.co/hardware/habana). To learn more about Hugging Face efforts to make AI hardware accelerators easy to use, check out our [Hardware Partner Program](https://huggingface.co/hardware). 
```

File: habana-gaudi-2-benchmark.md [hunk 7]
```diff
@@ -0,0 +1,144 @@
...
+    <a href="/regisss">
+        <img class="avatar avatar-user" src="https://aeiljuispo.cloudimg.io/v7/https://s3.amazonaws.com/moonup/production/uploads/1644920200150-620b7c408f5871b8a1a168a7.jpeg?w=200&h=200&f=face" title="Gravatar">
+        <div class="bfc">
+            <code>regisss</code>
+            <span class="fullname">Régis Pierrard</span>
+        </div>
+    </a>
+</div>
+
+In this article, you will learn how to start using [Habana Gaudi 2](https://habana.ai/training/gaudi2/) to accelerate model training and inference and train bigger models with 🤗 [Optimum Habana](https://huggingface.co/docs/optimum/habana/index). Then, several benchmarks will be presented to assess the performance gap between Gaudi 1, Gaudi 2 and Nvidia A100 80GB.
```
Comment (jeffboudier) [inline habana-gaudi-2-benchmark.md:25]:
```suggestion
In this article, you will learn how to use [Habana Gaudi 2](https://habana.ai/training/gaudi2/) to accelerate model training and inference, and train bigger models with 🤗 [Optimum Habana](https://huggingface.co/docs/optimum/habana/index). Then, we present several benchmarks including BERT pre-training, Stable Diffusion inference and T5 fine-tuning, to assess the performance gap between Gaudi 1, Gaudi 2 and Nvidia A100 80GB. Spoiler alert - Gaudi 2 is about twice faster than Nvidia A100 80GB for both training and inference!
```

Comment (jeffboudier) [reply habana-gaudi-2-benchmark.md:25]: the results are so cool I thought we should have the summary up top

**Review by jeffboudier — APPROVED** [review]
Super cool post!

File: habana-gaudi-2-benchmark.md (hunk 5 above)
Comment (regisss) [reply habana-gaudi-2-benchmark.md]: Good catch!

File: habana-gaudi-2-benchmark.md (hunk 7 above)
Comment (regisss) [reply habana-gaudi-2-benchmark.md]: Yes great idea!

**Pull request merged**